sdk/appconfiguration/azure-mgmt-appconfiguration/azure/mgmt/appconfiguration/_app_configuration_management_client.py

```diff
@@ -70,11 +73,18 @@ def __init__(
         self,
         credential: "TokenCredential",
         subscription_id: str,
-        base_url: str = "https://management.azure.com",
+        base_url: str = "",
+        *,
+        cloud_setting: Optional[AzureClouds] = None,
```

Comment:
Since most SDK users use public could, they don't care about this signature. Then we could put it in `kwagrs` to:
```
cloud = kwargs.pop("cloud_setting", None) or settings.current.azure_cloud
```